src/tokens/tokens.interfaces.ts

```diff
@@ -56,21 +95,23 @@ export class TokenPool {
   @IsNotEmpty()
   symbol: string;
 
+  @ApiProperty()
+  @IsNotEmpty()
+  type: TokenType;
+
   @ApiProperty({ description: contractConfigDescription })
   @IsOptional()
-  config?: any;
+  config: {
```

Comment:
If you annotate with `@IsOptional`, you should probably expect the item might be undefined. If it's required, you probably want to annotate with `@IsDefined` instead